documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -121,12 +121,26 @@ The default value is "PLAINTEXT".
 You can set the value to `PLAINTEXT`, `SSL`, `SASL_PLAINTEXT`, or `SASL_SSL`.
 
 . If you want to connect to Kafka brokers that are using certificates from a public certificate authority, set `STRIMZI_PUBLIC_CA` to `true`. Set this property to `true`, for example, if you are using Amazon AWS MSK service.
-. If you enabled TLS with the `STRIMZI_TLS_ENABLED` environment variable, specify the keystore and truststore used to authenticate connection to the Kafka cluster.
+. If you enabled TLS with the `STRIMZI_TLS_ENABLED` environment variable, specify the keystore and truststore (deprecated) or specify names of the secrets containing public and private keys used to authenticate connection to the Kafka cluster.
```

Comment:
Why do we specify secrets? Why not mount the certs into env var?

Reply:
The problem is that specifying just secret name enforces the secret structure on the user. That sucks. Mounting the files directly into env vars is on the other hand much more flexible. Doesn't depend how man secrets you have, what are the field names etc.

Reply:
I was not sure that is safe.

Reply:
It would also make sure the TO does not need any RBAC rights to read the secrets => the code would AFAIK not work right now?

Reply:
I did some manual testing and it worked. AFICS these [RBAC](https://github.com/strimzi/strimzi-kafka-operator/blob/main/packaging/install/cluster-operator/031-ClusterRole-strimzi-entity-operator.yaml#L36) rules are used

Reply:
But that is just because the user operator which works with secrets. The standalone TO does not have this RBAC rights.

Reply:
I think this needs to wait for https://github.com/strimzi/proposals/pull/46

Reply:
Then I guess we should maybe close it in the mean time? A long way before that is implemented.